File: core/src/main/java/com/ustadmobile/core/controller/SelectTwoDatesDialogPresenter.java
package com.ustadmobile.core.controller;

import java.util.Hashtable;

import com.ustadmobile.core.view.SelectTwoDatesDialogView;
import com.ustadmobile.core.impl.UstadMobileSystemImpl;


/**
 * The SelectTwoDatesDialog Presenter.
 */
public class SelectTwoDatesDialogPresenter
        extends UstadBaseController<SelectTwoDatesDialogView> {

    public SelectTwoDatesDialogPresenter(Object context, Hashtable arguments, SelectTwoDatesDialogView view) {
        super(context, arguments, view);
    }

    @Override
    public void onCreate(Hashtable savedState) {
        super.onCreate(savedState);
    }

    public void handleClickPrimaryActionButton() {
       view.finish();
    }

    @Override
    public void setUIStrings() {

    }

}

File: core/src/commonMain/kotlin/com/ustadmobile/core/controller/SelectTwoDatesDialogPresenter.kt
package com.ustadmobile.core.controller



import com.ustadmobile.core.view.SelectTwoDatesDialogView
import com.ustadmobile.core.impl.UstadMobileSystemImpl


/**
 * The SelectTwoDatesDialog Presenter.
 */
class SelectTwoDatesDialogPresenter(context: Any, arguments: Map<String, String>?, view:
SelectTwoDatesDialogView) : UstadBaseController<SelectTwoDatesDialogView>(context, arguments!!,
        view) {

    fun onCreate(savedState: Map<String, String>?) {
        super.onCreate(savedState)
    }

    fun handleClickPrimaryActionButton() {
        view.finish()
    }

}
